templates/zerver/help/send-timestamps.md

```diff
@@ -0,0 +1,26 @@
+# Send timestamps
+
+For communities distributed across the world, it is often difficult to
+convey time related information like meetings or events to everyone in
+chat. Zulip has a powerful feature to help: timestamps.
```

Comment:
When collaborating with people in another timezone, you often need to express a specific time clearly.  Rather than giving it in your timezone (or guessing that of the person you're talking to), in Zulip, you can mention a time, and it'll be displayed to each user in their own timezone (just like the timestamps on Zulip messages).